src/components/Incidents/IncidentsChart/IncidentsChart.jsx

```diff
@@ -1,111 +1,172 @@
 import * as React from 'react';
 
+import { ChartLegend, ChartBaseTheme } from '@patternfly/react-charts';
+import { Flex, FlexItem, Title } from '@patternfly/react-core';
+import './IncidentsChart.css';
+import IncidentsChartLegend from './IncidentsChartLegend';
+
 export function IncidentGanttChart({ groups, viewport }) {
-    // TODO useResizeObserver()
-    const width = 550;
-    const height = 100;
+  // TODO useResizeObserver()
+  const width = '100%';
+  const height = 200;
 
-    return (
-      <React.Component
-        style={{
-          position: 'relative',
-          width,
-          height,
-          overflow: 'hidden',
-          border: '1px solid #EFEFEF',
-        }}
+  const titleCss = {
+    fontSize: '--pf-v5-c-title--m-lg--FontSize',
+    fontWeight: '--pf-v5-c-title--m-lg--FontWeight',
+    lineHeight: '--pf-v5-c-title--m-lg--LineHeight',
+  };
+
+  return (
+    <div
+      style={{
+        position: 'relative',
+        width,
+        height,
+        overflow: 'hidden',
+        border: '1px solid #EFEFEF',
+        paddingBottom: '50',
+        paddingTop: '50',
+        paddingLeft: '50',
+        paddingRight: '50',
+      }}
+      className="incidents-top-chart-container"
```

Comment:
should we put the styling in CSS instead of inline if we already have a class for it?